docs/guides/manage-identities/15_customize-identity-schema.mdx

```diff
@@ -168,9 +149,9 @@ you want to know more about these types, please refer to the
 The `title` of each field is what the user as description or sample input. After adding the above examples the sign-up screen
 would look like so:
 
-![Example Identity Schema with GitHub and Phone Number Fields](/images/identity-model/example-identity0.png)
+![Example Identity Schema with GitHub and Phone Number Fields](../_static/manage-identities/2.png)
 
-### Change existing Identity Schemas
+### Change existing Identity schemas
```

Comment:
```suggestion
### Change existing Identity Schemas
```

So lets establish "Identity Schema" as proper ory specific term?

Reply:
Sorry, missed that.